.github/workflows/notify-docs-update.yaml

```diff
@@ -13,4 +13,4 @@ jobs:
     steps:
     - name: Send dispatch event
       run: |
-        curl -XPOST -u "${{ secrets.PAT_USERNAME }}:${{ secrets.PAT_TOKEN }}" -H "Accept: application/vnd.github.v3+json"  -H "Content-Type: application/json" https://api.github.com/repos/brimdata/zed-docs-site/dispatches --data '{"event_type":"zed-docs-update"}'
+        curl -XPOST -u "${{ secrets.PAT_USERNAME }}:${{ secrets.PAT_TOKEN }}" -H "Accept: application/vnd.github.v3+json" -H "Content-Type: application/json" https://api.github.com/repos/brimdata/superdb-website/dispatches --data '{"event_type":"super-docs-update", "client_payload": {"commit_sha": "${{ github.sha }}"}}'
```

Comment:
For now I've just got this rigged up to send the commits on this `hugo-docs` branch. When we eventually merge this to `main`, we can switch over to an approach like the one in [notify-downstream-merge.yaml](https://github.com/brimdata/super/blob/main/.github/workflows/notify-downstream-merge.yaml) that sends the merge commit SHA.